.github/workflows/createReleaseBranch.yml

```diff
@@ -64,3 +64,15 @@ jobs:
     - run: echo “Release Type is ${{ github.event.inputs.releaseType || 'minor' }}”
     - run: echo “Release Branch is ${{ steps.branch.outputs.branch }}”
 
```

Comment:
removed due to causing issues for the job this week.  Figured I'd put it back here since I was editing GHA files.